src/cli/commands/list/mod.rs

```diff
@@ -51,6 +55,11 @@ impl fmt::Display for ListOutput {
 }
 
 impl CommandExecution<ListOutput> for ListArgs {
+	/// This Command List and store all Cairo tests files within root directory.
+	/// Every file in Root directory must follow the following regex: "^test_.*\.cairo$"
```

Comment:
could you give an example for this regex:)?

Reply:
OK, some specific examples have been added. 
/// # Examples
	/// 	test_cairo_contracts/test_invalid_program.cairo => VALID
	/// 	test_hints/test/minor/failing.cairo => VALID
	/// 	test_cairo_hints/test_mock_call.cairo.test => INVALID, ends with "test" not ".cairo"
	/// 	mytest_hints/test_skip.cairo => INVALID, starts with "mytests" not "test_"